definitions/ext-service/golden_metrics.yml

```diff
@@ -0,0 +1,38 @@
+throughput:
+  title: Throughput
+  queries:
+    opentelemetry:
+      select: count(*)
+      from: Span
+      where: span.kind LIKE 'server' OR span.kind LIKE 'consumer' OR kind LIKE 'server' OR kind LIKE 'consumer'
+    kamon-agent:
+      select: sum(http.server.requests)
+      from: Metric
+    micrometer:
+      select: sum(http.server.requests)
+      from: Metric
+errorRate:
+  title: Error rate
+  queries:
+    opentelemetry:
+      select: filter(count(*), where error.message IS NOT NULL) / count(*)
+      from: Span
+    kamon-agent:
+      select: filter(sum(http.server.requests), where http.status_code = '5xx') / sum(http.server.requests)
+      from: Metric
+    micrometer:
+      select: filter(sum(http.server.requests), where exception IS NOT NULL and exception != 'None') / sum(http.server.requests)
+      from: Metric
+responseTimeMs:
+  title: Response time (ms)
+  queries:
+    opentelemetry:
+      select: average(duration.ms)
+      from: Span
```

Comment:
Do we know if spans are ok for Golden signals @naxhh ?

Reply:
I can't recall, i'm guessing no but ping @rcrocker

Reply:
They should be ok.

Reply:
Thanks!